tests/Alcaeus/MongoDbAdapter/Mongo/MongoClientTest.php

```diff
@@ -279,6 +279,14 @@ public function testConnectWithUsernameAndPasswordInConnectionUrl()
         $collection->insert($document);
     }
 
+    public function testConnectionUriOptionIntegerTypeCasting()
+    {
+        $client = new \MongoClient('mongodb://localhost/db?w=0&wtimeout=0', ['connect' => false]);
+
+        $this->skipTestIf(extension_loaded('mongo'));
```

Comment:
The test shouldn't be skipped: we want to ensure that the legacy driver and this adapter behave the same, which is achieved by running the test suite against both.

Reply:
I see, I've removed the skip line now.